Formula/kjell.rb

```diff
@@ -17,6 +17,11 @@ class Kjell < Formula
   depends_on "erlang"
 
   def install
+    # mkdir: $HOME/.kjell: Operation not permitted
+    # Prevent sandbox violation
+    # Reported 5th May 2016: https://github.com/karlll/kjell/issues/33
+    inreplace "Makefile", "CFG_DIR=~${USER}/.kjell", "CFG_DIR=#{pkgshare}"
```

Comment:
Could we make this just `${HOME}/.kjell`? That'll work in the sandbox and after installation by using different directories.